test/cookbooks/test/recipes/systemd_timer_resource.rb

```diff
@@ -2,6 +2,11 @@
   accept_chef_license true
   daemon_options ['--run-lock-timeout 0']
   environment 'FOO' => '1', 'BAR' => '2'
+  nice 19
```

Comment:
This is failing with the following:
```
undefined method `nice' for Chef::Resource::ChefClientSystemdTimer
```

Reply:
yeah because its testing on chef 16 which includes the resource and it takes precendence over the one in the cookbook. `chef_version_for_provides '< 16.0' if respond_to?(:chef_version_for_provides)` is in the resource which means the code in the cookbook is never truly tested and we are only testing the chef resource works. we may want to test on chef 15 so the cookbook resources also work.

FYI i had a chat with @tas50 and it looks like this doesn't do what you would expect because of forking/threading so i think i found a solution and will see to get it implemented if its ok with tim.

Not sure what to do about the tests running against chef 16 and not actually testing the cookbook code. open to ideas.